.github/workflows/llvm.yml

```diff
@@ -16,6 +16,10 @@ jobs:
       fail-fast: false
       matrix:
         llvm_version: ["13.0.0", "14.0.0", "15.0.6"]
+        include:
+          - llvm_version: "16.0.3"
+            force_version: true
```

Comment:
Do you happen to know why 16.0.3 is not available as a supported version that we need to force it?

Reply:
It is only unavailable from the [GitHub action that we use](https://github.com/KyleMayes/install-llvm-action)

Reply:
Should we submit a patch to add it then?

Reply:
All it does is fetch from llvm-project, which means the Windows installations do not have `llvm-config.exe`. For macOS we obtain LLVM via Nix. So not much is gained from this degree of portability

Reply:
I'm happy to revert to an explicit download and extraction if it improves things.